File: api/go/src/database/mysql.go
package database

import (
	"database/sql"
	"os"

	_ "github.com/go-sql-driver/mysql"
)

func GetConnectionMysql() (db *sql.DB, e error) {
	driver := "mysql"
	user := os.Getenv("MYSQL_USER")
	password := os.Getenv("MYSQL_PASSWORD")
	nombre := os.Getenv("MYSQL_NAME")
	host := os.Getenv("MYSQL_HOST")
	conexion, err := sql.Open(driver, user+":"+password+"@tcp("+host+")/"+nombre)
	if err != nil {
		return nil, err
	}
	return conexion, nil
}

File: backend/src/database/mysql.js
const mysql = require("mysql");
const util = require("util");

const pool = mysql.createPool({
  connectionLimit: 10,
  host: process.env.MYSQL_HOST,
  port: process.env.MYSQL_PORT,
  user: process.env.MYSQL_USER,
  password: process.env.MYSQL_PASSWORD,
  database: process.env.MYSQL_NAME,
});

pool.getConnection((err, connection) => {
  if (err) console.log("Something went wrong during connection Mysql");
  if (connection) {
    console.log("Mysql is connected to:", process.env.MYSQL_NAME);
    connection.release();
  }
  return;
});

pool.query = util.promisify(pool.query);

module.exports = pool;
